Automattic/woocommerce-payments
# Add connect account page experiment

Fixes #3790

#### Changes proposed in this Pull Request
- Add the experiment `woo_wcpayments_welcome_screen_cta_click_202203_v1` on connect account page (welcome screen) using `@woocommerce/explat`.
- Replicate the actual functionality and use the new provided design.

#### Testing instructions
While the experiment is created on our platform we need to manually switch between default and treatment components exchanging the following props:
https://github.com/Automattic/woocommerce-payments/blob/dea1682e7994bd273b430ef479eaf114973deafe/client/connect-account-page-experiment/index.js#L18-L19

- Without an account onboarded or with WCPay Dev `Force the plugin to act as disconnected from WCPay` enabled.
- Go to the connect screen on **WP Admin > Payments**
- You should see the new design for both mobile and desktop as on the screenshots below.
- Clicking on **Get Started** should redirect you to Stripe KYC flow.
- Go to **WP Admin > WooCommerce > Settings** and update your country to one unsupported by WCPay.
- Clicking on **Get Started** should show the non-supported modal.

#### Screenshots
| Desktop | Mobile | Modal |
|-|-|-|
|<img width="688" alt="Screenshot 2022-03-09 at 15 20 53" src="https://user-images.githubusercontent.com/7670276/157462018-926ba702-b259-42ee-9f36-099c46a6c25b.png">|<img width="348" alt="Screenshot 2022-03-09 at 15 21 10" src="https://user-images.githubusercontent.com/7670276/157462029-4cd3837c-4e31-45e8-a1ed-865d6fc521da.png">|<img width="463" alt="Screenshot 2022-03-09 at 15 41 55" src="https://user-images.githubusercontent.com/7670276/157464339-8627bc02-a835-4593-adca-e9f12072e4a4.png">|


-------------------


**Post merge**
